hvplot/__init__.py

```diff
@@ -1,3 +1,110 @@
+"""
+hvPlot makes data analysis and visualization simple
+===================================================
+
+hvPlot provides an easy to use, high-level, extended 🐼 Pandas .plot like API
+that works across a wide range of data sources and plotting backends.
+
+hvPlot
+
+- supports a wide range of data sources including Pandas, Dask, XArray
+Rapids cuDF, Streamz, Intake, Geopandas, NetworkX and Ibis.
+- supports the plotting backends Bokeh (default), Matplotlib and Plotly.
+- is build on top of HoloViews and allows you to drop into HoloViews when
+more flexibility and power is needed.
+
+To learn more check out https://hvplot.holoviz.org/. To report issues or contribute check out
+https://github.com/holoviz/hvplot. To join the community check out
+https://discourse.holoviz.org/.
+
+How to use hvPlot in 3 simple steps
+-----------------------------------
+
+Work with the data source you already know and ❤️
+
+>>> import pandas as pd, numpy as np
+>>> idx = pd.date_range('1/1/2000', periods=1000)
+>>> df  = pd.DataFrame(np.random.randn(1000, 4), index=idx, columns=list('ABCD')).cumsum()
+
+Import the hvplot extension for your data source
+
+>>> import hvplot.pandas
+
+Use the `.hvplot` api as you would use the Pandas `.plot` api.
+
+>>> curves=df.hvplot()
+>>> curves
+
+In a Jupyter Notebook, this will provide you with a plot of the
+A, B, C and D time series.
+
+For more check out the user guide https://hvplot.holoviz.org/user_guide/index.html
+
+How to select the kind of plot
+------------------------------
+
+You can add the `kind` argument
+
+>>> df.hvplot(kind="scatter")
+
+or use the specific kind of plot method
+
+>>> df.hvplot.scatter()
+
+For the available kinds of plots check out the reference gallery
+https://hvplot.holoviz.org/reference/index.html
+
+How to get help
+---------------
+
+To see the available arguments for a specific kind of plot run
+
+>>> import hvplot
+>>> hvplot.help(kind='scatter')
+
+In a notebook or ipython environment TAB and SHIFT+TAB completion will help you
+navigate and learn.
+
+How to export
+-------------
+
+>>> hvplot.save(curves, filename="curves.html")
+
+For more check out https://hvplot.holoviz.org/user_guide/Viewing.html#saving-plots
+
+How to drop into HoloViews
+--------------------------
+
+You don't have to do anything special as a hvPlot object is already a HoloViews
+object.
+
+You can understand the structure of your objects as usual by printing them
+
+>>> print(curves)
+:NdOverlay   [Variable]
+   :Curve   [index]   (value)
+
+and list the available options using `hv.help´.
+
+>>> import holoviews as hv
+>>> hv.extension("bokeh")
+>>> hv.help(curves)
+
+# Todo: fix https://github.com/holoviz/holoviews/issues/5364
+
+How to create data apps
```

Comment:
I find this part a little too specific for a high-level pitch, hvPlot users can live happily without knowing anything about Panel (the opposite holds too). Also it's suddenly serving a script while I tend to think the main usage of hvPlot is in a notebook for data exploration.

Maybe adding a similar section in a second step would be useful once hvPlot's documentation is improved, with a section that clearly explains how to create an app with Panel. The [Viewing](https://hvplot.holoviz.org/user_guide/Viewing.html) section does some of that but mixes different concepts.

Reply:
I feel strongly for this section. But its probably my personal preference. The reason why I put it in is 

1. I believe having a server with autoreload is a very efficient way of working that you as a user should now about. Its really a great starting point if you are trying to create something reusable.
2. Together with the HoloViews section is shows how you can drop into other parts of the ecosystem, such that there are no dead ends.

Maybe this is too much based on my experience and too much to add?